api/graphql/tests/snapshots/snap_test_reservation_units.py

```diff
@@ -79,12 +79,12 @@
                             {
                                 'begin': '2021-05-03T00:00:00+00:00',
                                 'end': '2021-05-03T01:00:00+00:00',
-                                'state': 'CREATED'
+                                'state': 'created'
```

Comment:
Just a note, we might have some inconsistency now between how the `state` is given in the input vs. how it appears in the output.

Currently you can filter reservations like this (state name in all caps):
```graphql
reservations(state: ["CREATED", "CONFIRMED"]) {
    begin
    end
    state
}
```
But in the output it will now appear in lowercase, like:
```js
'reservations': [{
    'begin': '2021-05-03T00:00:00+00:00',
    'end': '2021-05-03T01:00:00+00:00',
    'state': 'created',
}]
```
I'm not sure if using lowercase in the filters (`state: ["created", "confirmed"]`) works at the moment, but probably we should use either uppercase or lowercase consistently in both input and output.

Reply:
> Currently you can filter reservations like this (state name in all caps):

Actually checked that this isn't the case we cannot currently query reservations => I think we can add the filter for state in scope of its own ticket. So I'll focus on changing the mutation inputs to take the lower cases